packages/grafana-data/src/types/logsVolume.ts

```diff
@@ -0,0 +1,19 @@
+import { DataQuery } from './query';
+import { DataQueryRequest, DataQueryResponse } from './datasource';
+import { Observable } from 'rxjs';
+
+/**
+ * TODO: This should be added to ./logs.ts but because of cross reference between ./datasource.ts and ./logs.ts it can
+ * be done only after decoupling "logs" from "datasource" (https://github.com/grafana/grafana/pull/39536)
+ *
+ * @internal
+ */
+export interface DataSourceWithLogsVolumeSupport<TQuery extends DataQuery> {
+  getLogsVolumeDataProvider(request: DataQueryRequest<TQuery>): Observable<DataQueryResponse> | undefined;
+}
```

Comment:
@aocenas:

> There is no tooltip in the graph for some reason if you hover over the bar so not sure what the actual value is.

I get the tooltip even with a single bar:

<img width="1465" alt="Screen Shot 2021-09-30 at 09 07 51" src="https://user-images.githubusercontent.com/745532/135408767-be33af98-ce5c-4714-ad51-d5bfc4980fa0.png">

Could you send the response you got from Loki? I can try to mock it and see what happens

Reply:
Possibly related to running instant queries. This should work better with f5fc283dbe79d315d90fa3809239fdfea9488ac6 and will be revised in #39856.